server/plugin.go

```diff
@@ -40,7 +40,8 @@ type Plugin struct {
 	// templates are loaded on startup
 	templates map[string]*template.Template
 
-	ConfluenceClient *Client
+	serverVersionGreaterthan9 bool
+	ConfluenceClient          *Client
```

Comment:
`ConfluenceClient` doesn't seem to be in use in this PR:

```bash
$ ag ConfluenceClient
server/plugin.go
44:     ConfluenceClient          *Client
```

Reply:
removed